src/style/mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin size($width, $height) {
+  width: $width;
+  height: $height;
+}
+
+@mixin borderAndShadow($border, $radius, $shadow) {
+  border: $border;
+  border-radius: $radius;
+  box-shadow: $shadow; 
+}
```

Comment:
I like these mixins, particularly for shadows and shared styles across the app. Makes me think of some of the hawt storybook things that were happening in `crate`.